autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -169,6 +169,7 @@ def __init__(
                 # Tools are specified but overwrite_tools is False; do not update the assistant's tools
                 logger.warning("overwrite_tools is False. Using existing tools from assistant API.")
 
+        self.update_system_message(self._openai_assistant.instructions)
```

Comment:
Do we need to add this line?

Reply:
I found the system_message is empty while user create a GPTAssistantAgent using a existing OAI assistant ID. I think It's better to set it here, to avoid other issues.